processors/euctr/extractors.py

```diff
@@ -132,8 +132,15 @@ def extract_documents(record):
     if results_url:
         document = {
             'name': 'Results',
-            'type': 'results',
             'source_url': results_url,
         }
         documents.append(document)
     return documents
+
+
+def extract_document_category(record):
+    return {
+        'id': 24,
```

Comment:
`document_categories` will have to be kept in sink with Explorer's `data_categories` so we need a common identifier between them. If I choose `name` or [`name`, `group`] as `document_categories` unique identifier, when the `name` will change in `data_categories`, the corresponding `document_category` will not be updated but another one would be be created, leading to discrepancies. So I chose `data_categories` ID.